### PR TITLE
fix(RelativeRangeDatePicker): preset popup padding

### PR DESCRIPTION
**Pull request by saxumcordis**
corrected padding of popup with list..
before 
image
after
image

### CODE REVIEW

File: src/components/RelativeRangeDatePicker/components/Presets/Presets.scss [hunk 1]
```diff
@@ -11,7 +11,13 @@ $block: '.#{variables.$ns}relative-range-date-picker-presets';
         align-items: center;
         gap: var(--g-spacing-2);
 
+        padding: 0 var(--_--popup-content-padding);
```
Comment (ValeraS) [inline src/components/RelativeRangeDatePicker/components/Presets/Presets.scss:14]: let's add css variable, something like this
```scss
#{$block} {
    --g-list-item-padding: 0 var(--_--g-date-picker-presets-padding, 0);

    margin-inline: calc(-1 * var(--_--g-date-picker-presets-padding, 0));
    
    &__tabs {
        padding-inline: var(--_--g-date-picker-presets-padding, 0);
    }
}
```

and in `PickerDialog.scss` add:

```scss
#{$block} {
    &__content {
        --_--g-date-picker-presets-padding: var(--_--popup-content-padding);

        &_mobile {
            --_--g-date-picker-presets-padding: 0;
        }
    }
}
```

These seem to be all the changes needed.

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/RelativeRangeDatePicker/components/Presets/Presets.scss (hunk 1 above)
Comment (saxumcordis) [reply src/components/RelativeRangeDatePicker/components/Presets/Presets.scss:14]: Thank you!

but in mobile we shoudn't remove padding, i think
image
VS
image

Comment (ValeraS) [reply src/components/RelativeRangeDatePicker/components/Presets/Presets.scss]: Yes, I forgot to change the code in my comment to

```scss
    &__presets {
        margin-inline: calc(-1 * var(--_--g-date-picker-presets-padding));
    }
```

with it the mobile view will be the same as before.

**Comment by saxumcordis** [comment]
Sry, I do not know how to change status to"need review" :)

**Review by ValeraS — APPROVED** [review]

**Pull request merged**